var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -63,7 +63,7 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
         description="Ultra safety checks - " "used for increased robustness and self-diagnostics",
     )
     variant("ginkgo", default=False, description="Enable/disable ginkgo solver")
-    variant("cusolver", default=False, description="Enable/disable cuSovler")
+    variant("cusolver_lu", default=False, when="@0.7.1:", description="Enable/disable cuSovler")
 
```

Comment:
`cusolver_lu` variante enables very specific cuSolver feature, so it migth be a good idea to mention that specifically here.

```suggestion
    variant("cusolver_lu", default=False, when="@0.7.1:", description="Enable/disable cuSovler LU refactorization")

```